### PR TITLE
Add handling for inbox, and respond to comment replies

### PR DESCRIPTION
**Pull request by crhopkins**
Relevant issue: #42 
## Description:

Adds processing of inbox, replies to any comment replies to this bots comments.

### CODE REVIEW

File: tor_ocr/strings.py [hunk 1]
```diff
@@ -6,3 +6,8 @@
     'override whatever I provide, so please format my content accordingly '
     'if you choose to use it. '
 )
+
+reply_response = (
+    'Hello, I do not respond to commands. Perhaps you meant to reply to '
+    'u/transcribersofreddit instead.'
```
Comment (TimJentzsch) [inline tor_ocr/strings.py]: To make sure that this doesn't accidentally ping the other bot, we should probably escape the username with a backslash:
`r'u\/transcribersofreddit instead.'`

**Review by TimJentzsch — CHANGES_REQUESTED** [review]
I have a minor suggestion for improvement, otherwise this looks good!

**Comment by TimJentzsch** [comment]
There are also some minor formatting issues: https://github.com/GrafeasGroup/tor_ocr/runs/5364688789?check_suite_focus=true

**Comment by crhopkins** [comment]
> There are also some minor formatting issues: https://github.com/GrafeasGroup/tor_ocr/runs/5364688789?check_suite_focus=true

Yeah, I was wondering about that... I can add the spaces, I did it but thought it made things look worse. I saw in the TOR repo some lines done similarly that flake8 didn't flag, so I really wasn't sure.

**Comment by TimJentzsch** [comment]
Usually we just run the black formatting tool, `poetry run black .` should fix most formatting issues automatically.

**Review by TimJentzsch — APPROVED** [review]

**Comment by TimJentzsch** [comment]
Looks good now, thank you for your contribution!

**Pull request merged**